static/js/StaticPages.jsx

```diff
@@ -2636,6 +2636,222 @@ const StaticHR = () =>
+                    </h1>
+    
+                    {jobsAreAvailable ? (
+                        <>
+                            <h2>
+                                <span className="int-en">About Sefaria</span>
+                                <span className="int-he">אודות ספריא</span>
+                            </h2>
+                            <p>
+                                <span className="int-en">
+                                    Sefaria is a non-profit organization dedicated to creating the
+                                    future of Torah in an open and participatory way. We are assembling
+                                    a free, living library of Jewish texts and their interconnections,
+                                    in Hebrew and in translation.
+                                </span>
+                                <span className="int-he">
+                                    ספריא היא ארגון ללא מטרות רווח שמטרתו יצירת הדור הבא של לימוד התורה
+                                    באופן פתוח ומשותף. אנחנו בספריא מרכיבים ספרייה חיה וחופשית של טקסטים
+                                    יהודיים וכלל הקישורים ביניהם, בשפת המקור ובתרגומים.
+                                </span>
+                            </p>
+                        </>
+                    ) : null}
+                </header>
+            </>
+        );
+    };
+    
+    const Job = ({ job }) => {
+        return (
+            <div className="job">
+                <a className="joblink" target="_blank" href={job.jobLink}>
+                    {job.jobDescription}
+                </a>
+            </div>
+        );
+    };
+    
+    // Show the list of job postings within a department category
+    const JobsListForDepartment = ({ jobsList }) => {
+        return (
+            <section className="jobsListForDepartment">
+                {jobsList.map((job) => (
+                    <Job key={job.id} job={job} />
+                ))}
+            </section>
+        );
+    };
+    
+    // Job postings are grouped by department. This component will show the jobs for a specific department
+    // Each department has a header for its category before showing a list of the job postings there
+    const JobPostingsByDepartment = ({ department, departmentJobPostings }) => {
+        return (
+            <section className="section department englishOnly">
+                <header>
+                    <h2 className="anchorable">{department}</h2>
+                </header>
+                <JobsListForDepartment key={department} jobsList={departmentJobPostings} />
+            </section>
+        );
+    };
+    
+    // Show all the job postings, but first group them by department and render each department separately
+    const JobPostings = ({ jobPostings }) => {
+        const groupedJobPostings = jobPostings.reduce((jobPostingsGroupedByDepartment, jobPosting) => {
```

Comment:
I think we can do this bucketing already when retrieving the data initially, no? Seems a shame for there to be a data manipulation step in this component?